editor/inserter/menu.js

```diff
@@ -68,10 +68,15 @@ class InserterMenu extends wp.element.Component {
 	}
 
 	hoverBlock() {
+		const { lastMultiSelectedBlock, selectedBlock } = this.props;
+		let insertionPoint = null;
```

Comment:
Yes, right! Actually I tought about passing a prop from the `inserter` to the`menu` to void recomputing.